### PR TITLE
Hello Microverse Pull Request

### PR DESCRIPTION
**Pull request by Techcolony**
This is the Hello Microverse Pull Request to merge the main and feature-branch

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.header {
+  color: #291b1b;
+  background: #fff;
+}
```
Comment (mukhammadrizooff) [inline style.css]: - [ ] [REQUIRED] Good job so far! Kindly your style is not working, try to give style to your `h1` tag 👍🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Techcolony, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [x] [REQUIRED] Good job so far! Please add a descriptive summary of your `Pull Request` because `Pull Request` should have a summary describing introduced changes in more detail 🙏🏼 

image

- [x] [REQUIRED] Good job so far! `Pull Request` should have a descriptive title telling you what changes are inside kindly capitalize the first letter of your `PR` title🙏🏼 

- [x] [REQUIRED] README file should be descriptive please use [AKA: Microverse README](https://github.com/microverseinc/readme-template) template should be used 🙏🏼 

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Techcolony** [comment]
changes made

**Review by topeogunleye — APPROVED** [review]
### Hi @Techcolony, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
- [ ]  Next time, and for future commits, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Bad Commit Messages :x:
image

### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**